m365-developer-proxy/Program.cs

```diff
@@ -17,4 +17,41 @@
 
 rootCommand.Handler = proxyHost.GetCommandHandler(pluginEvents, loaderResults.UrlsToWatch, logger);
 
-return await rootCommand.InvokeAsync(args);
\ No newline at end of file
+// store the global options that are created automatically for us
+string[] globalOptions = { "--version", "-?", "-h", "--help" };
+
+// filter args to retrieve short (-n) and long (--name) option aliases
+var incomingOptions = args.Where(arg => arg.StartsWith("-") || arg.StartsWith("--")).ToArray();
```

Comment:
We can simplify this. An arg that starts with `--` also starts with `-` so the first check will already capture the second, which we can remove. Here, we should also consider the Windows-style switches with `/`

Reply:
Done